### ISSUE TITLE
weird eval page that shouldn't exist: https://docs.gravwell.io/scripting/eval.html

### ISSUE THREAD

**Issue by kris-watts-gravwell** [gravwell/wiki#831]
this page shouldn't exist: https://docs.gravwell.io/scripting/eval.html

**Comment by ashnwade** [comment]
We also have anko under automation (same as the eval module page you asked to remove):
![Image](https://github.com/gravwell/wiki/assets/107071905/a0104444-83a2-430a-a90c-6f6e3e09020a)

And an anko module documented under processing modules (same as eval):
![Image](https://github.com/gravwell/wiki/assets/107071905/bdd6f038-cbb0-4ef2-8222-f79ab9a17374)

Should anko be documented exclusively under processing modules as well? See [the Scripting Overview](https://docs.gravwell.io/scripting/scripting.html) where both are referenced.

**Comment by kris-watts-gravwell** [comment]
Correct, i think [this](https://docs.gravwell.io/scripting/scriptingsearch.html) covers the anko scripting piece, both "eval module" and "anko module" should probably be removed.
